pkg/core/xds/v3/context.go

```diff
@@ -0,0 +1,67 @@
+package v2
```

Comment:
Do you think it makes sense to keep this `pkg/core/xds` package? It'd be nice to get rid of `core` package sometime in the future, maybe it's possible to just copy this `pkg/core/xds` into `pkg/xds`? It doesn't have to be the scope of this PR, just wondering about our plans

Reply:
discussed offline.